### PR TITLE
[4.0] [com_menus] Untranslated string

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #1043 .

### Zusammenfassung der Änderungen

add new string
translation
punctuation marks adapted

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

backend

### CODE REVIEW

File: administrator/language/de-DE/com_menus.ini [hunk 1]
```diff
@@ -120,20 +120,21 @@ COM_MENUS_ITEMS_REBUILD_FAILED="Fehler beim Wiederherstellen der Liste der Menü
 COM_MENUS_ITEMS_REBUILD_SUCCESS="Liste der Menüeinträge wiederhergestellt"
 COM_MENUS_ITEMS_SEARCH_FILTER="Suche nach einem Titel oder Alias. Bei Verwendung eines Präfixes mit „ID: “ erfolgt die Suche nach einer Menü-ID"
 COM_MENUS_ITEMS_SEARCH_FILTER_LABEL="Menüeinträge durchsuchen"
-COM_MENUS_ITEMS_SET_HOME_1="Ein Menüeintrag wurde als Startseite ausgewählt!"
-COM_MENUS_ITEMS_SET_HOME_MORE="%d Menüeinträge wurden als Startseite ausgewählt!"
+COM_MENUS_ITEMS_SET_HOME_0="Kein Menüeintrag wurde als Startseite ausgewählt."
```
Comment (zero-24) [inline administrator/language/de-DE/com_menus.ini:123]:
```suggestion
COM_MENUS_ITEMS_SET_HOME_0="Es wurde kein Menüeintrag als Startseite ausgewählt."
```

File: administrator/language/de-DE/com_menus.ini [hunk 2]
```diff
@@ -120,20 +120,21 @@ COM_MENUS_ITEMS_REBUILD_FAILED="Fehler beim Wiederherstellen der Liste der Menü
...
+COM_MENUS_ITEMS_SET_HOME_1="Ein Menüeintrag wurde als Startseite ausgewählt."
```
Comment (zero-24) [inline administrator/language/de-DE/com_menus.ini:124]:
```suggestion
COM_MENUS_ITEMS_SET_HOME_1="Es wurde ein Menüeintrag als Startseite ausgewählt."
```

**Comment by tecpromotion** [comment]
@zero-24 with the proposals one would also have to adapt line https://github.com/joomlagerman/joomla/blob/c387dd9b53c7b1776fed43c9bdd0cb0e5fef6c3c/administrator/language/de-DE/com_menus.ini#L127, right?

File: administrator/language/de-DE/com_menus.ini [hunk 3]
```diff
@@ -120,20 +120,21 @@ COM_MENUS_ITEMS_REBUILD_FAILED="Fehler beim Wiederherstellen der Liste der Menü
...
+COM_MENUS_ITEMS_SET_HOME_MORE="%d Menüeinträge wurden als Startseite ausgewählt."
 COM_MENUS_ITEMS_TABLE_CAPTION="Tabelle der Menüeinträge"
-COM_MENUS_ITEMS_UNSET_HOME="Ein Menüeintrag wurde als Startseiteneintrag zurückgesetzt!"
+COM_MENUS_ITEMS_UNSET_HOME="Ein Menüeintrag wurde als Startseiteneintrag zurückgesetzt."
```
Comment (zero-24) [inline administrator/language/de-DE/com_menus.ini:127]:
```suggestion
COM_MENUS_ITEMS_UNSET_HOME="Es wurde ein Menüeintrag als Startseiteneintrag zurückgesetzt."
```

**Comment by zero-24** [comment]
> @zero-24 with the proposals one would also have to adapt line

Right, Just added that

File: administrator/language/de-DE/com_menus.ini (hunk 3 above)
Comment (tecpromotion) [reply administrator/language/de-DE/com_menus.ini:127]:
```suggestion
COM_MENUS_ITEMS_UNSET_HOME="Es wurde ein Menüeintrag wurde als Startseite zurückgesetzt."
```

Comment (tecpromotion) [reply administrator/language/de-DE/com_menus.ini:127]: sounds nicer

**Pull request merged**